packages/ui-scripts/lib/icons/generate-icon-fonts.js

```diff
@@ -41,9 +41,8 @@ export default function generateIconFonts({
   name
 }) {
   generateFonts({ ...config, inputDir, outputDir, prefix, name })
-    .then((result) => {
-      // eslint-disable-next-line no-console
-      console.log('Icon font generated:', result)
+    .then((_result) => {
+      //console.log('Icon font generated:', result)
```

Comment:
This is generating the 'icon created' spam on the beginning of `npm run boorstrap` I've turned this off since its generating like 5000 line of logs